src/bloqade/task/base.py

```diff
@@ -95,9 +109,57 @@ def run(self, **kwargs):
         raise NotImplementedError
 
 
-# Report is now just a helper class for
-# organize and analysis data:
 class Report:
+    """Report is a helper class for organizing and analysing data
+
+    ### Analyzing Results using `Report` object
+    When you've retrieved your results from either emulation
+    or hardware you can generate a `.report()`:
+
+    ```python
+    report = results.report()
+    ```
+
+    For the examples below we analyze the results of a two atom program.
+
+    The report contains useful information such as:
+
+    * The raw bitstrings measured per each execution of the program
+    ```python
+    report.bitstrings()
+    ```
```

Comment:
just use the terminal style in one code block? e.g

```suggestion
    ```python
    >>> report.bitstrings()
    [array([[1, 1],
            [1, 1],
            [1, 1],
            ...,
            [1, 1],
            [1, 1],
            [1, 0]], dtype=int8)]
    ```
```